### PR TITLE
Fixed PBO WCA Logic

### PR DESCRIPTION
**Pull request by Ollieollieolllie**

### CODE REVIEW

File: @AH-64D Apache Longbow/Addons/fza_ah64_wca/functions/fn_update.sqf [hunk 1]
```diff
@@ -21,10 +21,12 @@ params ["_heli"];
 
 private _onGnd      = isTouchingGround _heli;
 private _gndOrideOn = _heli getVariable "fza_ah64_gndOrideOn";
+private _battBusOn = _heli getVariable "fza_systems_battBusOn";
 
-if (!_gndOrideOn && _onGnd) then {
+if (!_gndOrideOn && _onGnd || !_battBusOn) then {
```
Comment (BradMick) [inline @AH-64D Apache Longbow/Addons/fza_ah64_wca/functions/fn_update.sqf:26]: None of this should function if the battery isn't on, should move this up above and use an exitWith

File: @AH-64D Apache Longbow/Addons/fza_ah64_wca/functions/fn_wcaBtnHandleControl.sqf [hunk 2]
```diff
@@ -20,13 +23,13 @@ switch (_control) do {
         playsound "fza_ah64_switch_flip4";
     };
     case "mstrCaut": {
-        _heli setVariable ["fza_ah64_mstrCautLightOn", !(_heli getVariable "fza_ah64_mstrCautLightOn"), true];
+        _heli setVariable ["fza_ah64_mstrCautLightOn", false, true];
```
Comment (BradMick) [inline @AH-64D Apache Longbow/Addons/fza_ah64_wca/functions/fn_wcaBtnHandleControl.sqf:26]: why did you change this if you didn't change it everywhere else? 

we either need to use use:

!(_heli getVariable "fza_ah64_emerHydOn")

or explicitly set everything to false as you've done here

File: @AH-64D Apache Longbow/Addons/fza_ah64_wca/functions/fn_update.sqf (hunk 1 above)
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_wca/functions/fn_update.sqf:26]: thought it was simpler, to include in the if statement, that duplicating the output

File: @AH-64D Apache Longbow/Addons/fza_ah64_wca/functions/fn_wcaBtnHandleControl.sqf (hunk 2 above)
Comment (Ollieollieolllie) [reply @AH-64D Apache Longbow/Addons/fza_ah64_wca/functions/fn_wcaBtnHandleControl.sqf:26]: can you toggle master warning/caution light on/off, because thats whats happeniing

Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_wca/functions/fn_wcaBtnHandleControl.sqf:26]: Nope, your change here is good, didn't catch onto the issue at first.

File: @AH-64D Apache Longbow/Addons/fza_ah64_wca/functions/fn_update.sqf (hunk 1 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_wca/functions/fn_update.sqf:26]: This'll work since it's the only thing being called.

**Review by BradMick — APPROVED** [review]
